packages/back-end/src/main.ts

```diff
@@ -155,7 +155,7 @@ if (process.env.CI_CD === 'true') {
         systemAdminPassword: systemAdminPassword,
         testUserEmail: testUserEmail,
         testUserPassword: testUserPassword,
-        seqeraApiBaseUrl: seqeraApiBaseUrl,
+        seqeraApiBaseUrl: seqeraApiBaseUrl.replace(/\/+$/, ''), // Remove trailing slashes
```

Comment:
Could you just use trim here?

Reply:
@PurcyTwoBulls , sorry no trim is just for removing whitespaces. This preventive fix is to remove any potential trailing slashes as we already insert a `'/'` when we prepare the NextFlow API URLs.

Additionally, we explicitly document that the values in the `easy-genomics.yaml` file be enclosed in quotes to prevent whitespaces.